packages/@cdk-cosmos/core/src/solar-system.ts

```diff
@@ -83,6 +87,14 @@ export class SolarSystemCoreStack extends BaseStack implements ISolarSystemCore
       zoneName: `${id}.${rootZone.zoneName}`.toLowerCase(),
       comment: `Core Main Zone for ${id} SolarSystem`,
     });
+    if (cert) {
+      this.certificate = new DnsValidatedCertificate(this, 'Certificate', {
```

Comment:
do we need to be able to consume the cert from an extension ? (export and import ?)

Reply:
Dont see any need for now